packages/react-is/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "react-is",
+  "description": "Brand checking of React Elements.",
+  "keywords": [
+    "react"
+  ],
+  "version": "1.0.0",
+  "homepage": "https://reactjs.org/",
+  "bugs": "https://github.com/facebook/react/issues",
+  "license": "MIT",
+  "files": [
+    "index.js"
+  ],
+  "main": "index.js",
+  "repository": "facebook/react",
+  "engines": {
+    "node": ">=0.10.0"
```

Comment:
We support Node 0.10?

Reply:
Copypasta from https://github.com/facebook/react/blob/master/packages/react/package.json#L20-L22

I guess we don't include that in other packages..?